.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
+npmRegistryServer: https://registry.npmjs.org
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
+yarnPath: .yarn/releases/yarn-3.6.4.cjs
+npmPublishAccess: public
```

Comment:
By default, scoped packages are private. Since this is public, we need to specify it explicitly here.